packages/phrases/src/locales/fr/errors.ts

```diff
@@ -67,6 +67,8 @@ const errors = {
     general: "Une erreur inattendue s'est produite dans le connecteur. {{errorDescription}}",
     not_found: 'Impossible de trouver un connecteur disponible pour le type : {{type}}.',
     not_enabled: "Le connecteur n'est pas activé.",
+    invalid_metadata: "The connector's metadata is invalid.", // TODO: need to check FR translation
```

Comment:
good comment, maybe we can use a common token instead of `// TODO`? e.g. `// UNTRANSLATED`. then contributors can easily find them